content/id/docs/tasks/administer-cluster/manage-resources/memory-constraint-namespace.md

```diff
@@ -0,0 +1,272 @@
+800 MiB. Nilai tersebut memenuhi batasan yang ditentukan oleh LimitRange.
+
+```yaml
+resources:
+  limits:
+     memory: 800Mi
+  requests:
+    memory: 600Mi
+```
+
+Menghapus Podmu:
+
+```shell
+kubectl delete pod constraints-mem-demo --namespace=constraints-mem-example
+```
+
+## Mencoba membuat Pod yang melebihi batasan maksimum memori
+
+Berikut berkas konfigurasi untuk sebuah Pod yang memiliki satu Kontainer. Kontainer tersebut menentukan
+permintaan memori 800 MiB dan batas memori 1.5 GiB.
+
+{{< codenew file="admin/resource/memory-constraints-pod-2.yaml" >}}
+
+Mencoba membuat Pod:
+
+```shell
+kubectl apply -f https://k8s.io/examples/admin/resource/memory-constraints-pod-2.yaml --namespace=constraints-mem-example
+```
+
+Keluaran yang dihasilkan menunjukkan Pod tidak dibuat, karena Kontainer menentukan limit memori yang
+terlalu besar:
+
+```
+Error from server (Forbidden): error when creating "examples/admin/resource/memory-constraints-pod-2.yaml":
+pods "constraints-mem-demo-2" is forbidden: maximum memory usage per Container is 1Gi, but limit is 1536Mi.
+```
+
+## Mencoba membuat Pod yang tidak memenuhi permintaan memori
+
+Berikut berkas konfigurasi untuk sebuah Pod yang memiliki satu Kontainer. Kontainer tersebut menentukan
+permintaan memori 100 MiB dan limit memori 800 MiB.
+
+{{< codenew file="admin/resource/memory-constraints-pod-3.yaml" >}}
+
+Mencoba membuat Pod:
+
+```shell
+kubectl apply -f https://k8s.io/examples/admin/resource/memory-constraints-pod-3.yaml --namespace=constraints-mem-example
+```
+
+Keluaran yang dihasilkan menunjukkan Pod tidak dibuat, karena Kontainer menentukan permintaan memori yang
+terlalu kecil:
+
+```
+Error from server (Forbidden): error when creating "examples/admin/resource/memory-constraints-pod-3.yaml":
+pods "constraints-mem-demo-3" is forbidden: minimum memory usage per Container is 500Mi, but request is 100Mi.
+```
+
+## Membuat Pod yang tidak menentukan permintaan ataupun limit memori
+
+
+
+Berikut berkas konfigurasi untuk sebuah Pod yang memiliki satu Kontainer. Kontainer tersebut tidak menentukan
+permintaan memori dan juga limit memori.
+
+{{< codenew file="admin/resource/memory-constraints-pod-4.yaml" >}}
+
+Mencoba membuat Pod:
+
+```shell
+kubectl apply -f https://k8s.io/examples/admin/resource/memory-constraints-pod-4.yaml --namespace=constraints-mem-example
+```
+
+Melihat informasi mendetail tentang Pod:
+
+```
+kubectl get pod constraints-mem-demo-4 --namespace=constraints-mem-example --output=yaml
+```
+
+Keluaran yang dihasilkan menunjukkan Kontainer pada Pod memiliki permintaan memori 1 GiB dan limit memori 1 GiB.
+Bagaimana Kontainer mendapatkan nilai tersebut?
+
+```
+resources:
+  limits:
+    memory: 1Gi
+  requests:
+    memory: 1Gi
+```
+
+Karena Containermu tidak menentukan permintaan dan limit memori, Kontainer tersebut diberikan
+[permintaan dan limit memori semula](/docs/tasks/administer-cluster/manage-resources/memory-default-namespace/)
```

Comment:
```suggestion
[permintaan dan limit memori bawaan](/docs/tasks/administer-cluster/manage-resources/memory-default-namespace/)
```

Reply:
Resolved in e323b8c5920d6ae02c050306495e784b8e19ad9f